python/main.py

```diff
@@ -85,13 +93,8 @@ def getpayee(payeename):
             source="Imported"
         )
 
-        if not ynab_client.budget.be_transactions.containsduplicate(transaction):
-            log.debug('Appending transaction %s '%transaction.getdict())
-            transactions.append(transaction)
-        else:
-            log.debug('Duplicate transaction found %s '%transaction.getdict())
-
-        ynab_client.add_transactions(transactions)
+        ynab_client.budget.be_transactions.append(transaction)
```

Comment:
@rossdargan sorry for the delay. Are we ok with getting rid of duplicate checking here? We needed it for webhook retrying.